projects/requisition-management/src/app/pages/requisition-detail/requisition-detail-page.component.html

```diff
@@ -115,13 +115,13 @@ <h3>{{ 'checkout.order_summary.heading' | translate }}</h3>
     ></ish-requisition-reject-dialog>
 
     <div class="section d-flex d-flex justify-content-between">
-      <a [routerLink]="['../../' + view, { status: requisition.approval.statusCode }]">
+      <a [routerLink]="['../../' + view, { status: requisition.approval.statusCode }]" class="noprint">
```

Comment:
Please rename the class to `no-print` (and change at all places in HTML an SCSS.